packages/manager/modules/pci/src/projects/project/instances/instance/reinstall/reinstall.html

```diff
@@ -18,4 +18,13 @@
             data-translate="pci_projects_project_instances_instance_reinstall_erase_message"
         ></span>
     </oui-message>
+
+    <oui-message
```

Comment:
As a customer I will see a modal like this : 

----
**Re-install an instance**
----
Are you sure that you want to re-install the {{ instance }} instance?

:warning:  &nbsp; All of the data stored on it will be deleted.

:warning: &nbsp; L'image de votre instance est dépréciée. Vous allez être redirigé vers la page d'édition.

[ cancel | **confirm** ]

----

Not sure it's easy to understand

Reply:
@cbourgois, do you have a suggestion ?

Reply:
@mohammed-zahaf i think you could simply display a single message saying the instance image is deprecated and that the customer need to choose another image. Then instead of a "confirm" button in the modal you should display something like "Edit instance image" that will perform the redirection.
Anything that can improve the user experience and make the process more understandable is welcome

Reply:
Thanks @frenautvh for suggestion.
i got a new content.